src/pages/admin/regulatory/forms/CreateRegulatoryForm.js

```diff
@@ -38,7 +38,7 @@ const CreateRegulatoryForm = props => {
     setFieldTouched(name, true, false);
   };
 
-  const regulatoryAttch = { name : "", testName : "", releaseTime : "", requirement : "", averageRelease : "", stage : ""};
+  const attachment = { docName : "", description : "", mandatory : ""};
```

Comment:
This represents your attachment row. Once you click on add new we are pushing this attachment object to attachments(notice the 's') object array. Pushing part I will mention below